test/unzipObjWith.js

```diff
@@ -0,0 +1,70 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+
+describe('unzipObjWith', function() {
+  it('unzips an object into key/value arrays applying a value/key transform', function() {
+    const obj = {
```

Comment:
Pls simplify the data inside tests. The tests are so complicated here it's not apparent what is happening.